openapi-bal-service/src/main/java/io/ballerina/openapi/converter/utils/ConverterCommonUtils.java

```diff
@@ -522,4 +526,17 @@ public static String unescapeIdentifier(String parameterName) {
         String unescapedParamName = IdentifierUtils.unescapeBallerina(parameterName);
         return unescapedParamName.replaceAll("\\\\", "").replaceAll("'", "");
     }
+
+    public static void setDefaultValue(Schema schema, String defaultValue) {
+        if (Objects.isNull(schema) || Objects.isNull(defaultValue)) {
+            return;
+        }
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.configure(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES, true);
+            schema.setDefault(mapper.readValue(defaultValue, Object.class));
+        } catch (JsonProcessingException e) {
```

Comment:
What are the possible scenarios this JsonProcesingException can happen?

Reply:
The exception is thrown if the the string does not match the JSON string format. (Here we have allowed the json fields without quotes to support ballerina record literals)
So there should not be a case where the literal does not align with JSON format. This try-catch has been added as a saftly mechanism.

Reply:
What is the `defaultValue` value in such cases, when the exception occur ?

Reply:
It is just the ballerina literal expression, in those exceptional cases we just print that expression as string. But ideally this exception will never happen